packages/node/src/index.ts

```diff
@@ -454,6 +454,8 @@ export const build: BuildV3 = async ({
       entrypoint: handler,
       files: preparedFiles,
 
```

Comment:
This needs to be added to every builder that calls `new EdgeFunction()`:

- `@vercel/node`
- `@vercel/next`
- `@vercel/hydrogen`
- `@vercel/static-build`